tun2socks/tun2socks.c

```diff
@@ -485,11 +489,13 @@ int main (int argc, char **argv)
         goto fail0;
     }
 
+#ifdef __ANDROID__
     if (options.fake_proc) {
         // Fake process name to cheat on Lollipop
-        strcpy(argv[0], "com.github.shadowsocks");
-        prctl(PR_SET_NAME, "com.github.shadowsocks");
+        strcpy(argv[0], ANDROID_APP_PACKAGE_NAME);
+        prctl(PR_SET_NAME, ANDROID_APP_PACKAGE_NAME);
     }
+#endif
```

Comment:
I believe neither of these are being used by shadowsocks-android latest master.

Reply:
I have tested it in Linux, it will error if not check `__ANDROID__`.

Reply:
I was talking about `com.github.shadowsocks`. Maybe it is appropriate to remove these two altogether.